CHANGELOG.md

```diff
@@ -4,5 +4,15 @@
 
 # v1.0.0 / 2021-07-29 [(commits)](https://github.com/fastruby/dotenv_validator/tree/v1.0.0)
 
+# v1.1.0 /2021-09-14 [(commits)](https://github.com/fastruby/dotenv_validator/compare/v1.0.0...v1.1.0)
+  - [Implement overcommit + styleguide](https://github.com/fastruby/dotenv_validator/pull/18)
+  - [Add sponsorship section and logo to the README](https://github.com/fastruby/dotenv_validator/pull/19)
```

Comment:
wondering if something like this is really relevant for the changelog, since it doesn't really affect the gem

Reply:
A changelog should document all the changes in that new version regardless of the impact is having on the project